packages/core/components/ui/Icon.tsx

```diff
@@ -88,15 +88,24 @@ type IconProps = {
   /* Returns icon data as plain svg */
   base?: boolean
   /* Sets alt text for the icon */
-  alt: string
+  alt?: string
```

Comment:
We might want to keep this requirement for 508.

Reply:
I think having a different description of a checkmark throughout the app might be worse than defaulting to checkmark. I could create a new lookup with default names for each of the icons? so it doesn't end up as 'caretUp'. 

`const defaultNameLookup = { caretUp: 'caret up'}`

`<IconObj title={ alt || defaultNameLookup[display] } />`

if you think this is necessary. I just kept seeing a lot of places where the alt wasn't given and we had typescript errors.